compiler/rustc_data_structures/src/profiling.rs

```diff
@@ -705,15 +731,21 @@ impl<'a> TimingGuard<'a> {
 
 #[must_use]
 pub struct VerboseTimingGuard<'a> {
-    start_and_message: Option<(Instant, Option<usize>, String)>,
+    start_and_message: Option<(Instant, Option<usize>, String, TimePassesFormat, bool)>,
```

Comment:
With five elements, I think this tuple is now big enough that it should have its own type. That would allow some comments, particularly for the `bool` field. I see it's called `unique`, but at this point in the review it's not clear to me what that means.

Reply:
Also, `is_unique` might be a better name? Or a whole new 2-value enum might be nicer than just `bool`.